docs/pipelines_yaml_files.md

```diff
@@ -124,7 +127,9 @@ Indicates that this pipeline can be used for requests with a request type of 'li
 
 The most common keys to filter on are request_type and library_type.
 
-All filters must be fulfilled for a pipeline to be considered valid.
+All filters must be fulfilled for a pipeline to be considered valid for the specific instance of Labware in question. Note that the labware must also have a purpose that matches to one of those listed in the relationships section of the pipeline yaml. This is partly why we now try to prefix purpose keys with pipeline specific characters, use request types specific to pipeline submission templates, and use library types specific to their library prep pipelines where ever possible.
```

Comment:
`Note that the labware must also have a purpose that matches to one of those listed in the relationships section of the pipeline yaml.`

This bit is pretty important. Maybe this note could also be added to the other pipeline/ purpose docs

Reply:
Ok will do.